docs/UserGuide.md

```diff
@@ -15,20 +15,25 @@ interactions, FitBook is the perfect application for you to manage your clients!
 ## Quick Start
 
 1. Ensure you have Java `11` or above installed.
+    * Need help? Check out the [installation guide](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html).
 
-2. Download the latest version of `FitBook.jar` from [here](https://github.com/AY2324S2-CS2103T-T17-3/tp/releases).
+1. Download the latest version of `FitBook.jar` from [here](https://github.com/AY2324S2-CS2103T-T17-3/tp/releases).
 
-3. Copy the file to the folder you want to use as the **home folder** for FitBook.
+1. Copy the file to the folder you want to use as the **home folder** for FitBook.
 
-4. Double-click the FitBook.jar file to launch it
+1. Double-click the FitBook.jar file to launch it.
+    * If that doesn't work, try the following steps:
 
-5. If (4) doesn't work, open a command terminal, `cd` into the folder with FitBook.jar, and run the 
-`java -jar FitBook.jar` command to launch the application.<br>
+        1. Open a command prompt (**cmd** for Windows, **Terminal** for Mac)
```

Comment:
Since we are officially supporting it, would it be good to include an example for Linux as well? 

Although it might be troublesome as different Linux distros ship with different default terminal emulators

Reply:
I considered it, but decided not to as Linux users likely know what a command prompt is and how to use the command prompt.